### ISSUE TITLE
Inefficient Regular Expression Complexity in moment

### ISSUE THREAD

**Issue by landsman** [DevExpress/testcafe#7163]
### What is your Scenario?

We have a report on security vulnerability thanks to Github in our private repository.
I want to get rid of it.

I found that's because of the dependency used in `testcafe` package.

```
landsman@M1 pay % yarn why moment
├─ testcafe-legacy-api@npm:5.1.4
│  └─ moment@npm:2.29.2 (via npm:^2.14.1)
│
└─ testcafe@npm:1.19.0
   └─ moment@npm:2.29.4 (via npm:^2.29.3)
```

https://github.com/advisories/GHSA-wc69-rhjr-hc9g

### What is the Current behavior?

security vulnerability

### What is the Expected behavior?

no security vulnerability

### What is your public website URL? (or attach your complete example)

no

### What is your TestCafe test code?

no

### Your complete configuration file

no

### Your complete test report

no

### Screenshots

no

### Steps to Reproduce

https://github.com/advisories/GHSA-wc69-rhjr-hc9g

### TestCafe version

1.19.0

### Node.js version

no

### Command-line arguments

no

### Browser name(s) and version(s)

no

### Platform(s) and version(s)

no

### Other

Anyway, I was not able to submit this form even if you are saying that the required fields are marked with `*`.
I had to fill them all to unblock the disabled button `Submit new issue`. Weird.

**Comment by miherlosev** [comment]
Hi @landsman,

I was unable to reproduce this issue.
![image](https://user-images.githubusercontent.com/4133518/179463022-36b2ea07-e1cc-4327-8051-0d48c95749e6.png)

Please remove the `yarn.lock` file and perform the clean module installation.

**Comment by landsman** [comment]
Check this out:

```
landsman@M1 pay % yarn why moment
├─ testcafe-legacy-api@npm:5.1.4
│  └─ moment@npm:2.29.2 (via npm:^2.14.1)
│
└─ testcafe@npm:1.20.0
   └─ moment@npm:2.29.4 (via npm:^2.29.4)
landsman@M1 pay % yarn why testcafe-legacy-api
└─ testcafe@npm:1.20.0
   └─ testcafe-legacy-api@npm:5.1.4 (via npm:5.1.4)
```

**Comment by landsman** [comment]
Okay, my bad.
I tried to remove `testcafe` from the project and install it again. Now it's fixed.

```
landsman@M1 pay % yarn why moment
├─ testcafe-legacy-api@npm:5.1.4
│  └─ moment@npm:2.29.4 (via npm:^2.14.1)
│
└─ testcafe@npm:1.20.0
   └─ moment@npm:2.29.4 (via npm:^2.29.4)
```

**Comment by miherlosev** [comment]
Good. I am happy to hear that the issue is resolved.
